docs/interactions/Slash_Commands.md

```diff
@@ -696,18 +696,18 @@ Fetch a global command for your application. Returns an [ApplicationCommand](#DO
 ## Edit Global Application Command % PATCH /applications/{application.id#DOCS_TOPICS_OAUTH2/application-object}/commands/{command.id#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommand}
 
 > info
-> All parameters for this endpoint are optional. `options` is nullable.
+> All parameters for this endpoint are optional.
```

Comment:
I think it would be better to just remove this, and have `?` on each parameter.

Reply:
This info block is common for PATCH endpoints